.github/workflows/test.yml

```diff
@@ -0,0 +1,80 @@
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - 'releases/*'
+
+jobs:
+  build_test_job:
+    name: 'Build and test job'
+    runs-on: ${{ matrix.os }}
+    strategy:
+        matrix:
+          os: [windows-latest, ubuntu-latest, macos-latest]
+    steps:
+
+    - name: 'Checking out repo code'
+      uses: actions/checkout@v1
+
+    - name: Extract branch name
+      id: extract_branch
+      run: |
+        echo "##[set-output name=branchname;]$(echo ${GITHUB_REF##*/})"
```

Comment:
What's the need for this?

Reply:
If this is required.. then add the `github.event.pull_request.base.ref` part to this as well and have a single source for the branch name